src/move.cpp

```diff
@@ -2026,6 +2026,7 @@ static void checkLocalFeatures(DROID *psDroid)
 
 	// scan the neighbours
 #define DROIDDIST ((TILE_UNITS*5)/2)
+#define PICKUPZDIST (64)
```

Comment:
Since we are C++14 now, let's use proper `constexpr` constants (and a bit more descriptive name):
```suggestion
constexpr int MAX_PICKUP_DISTANCE = 64;
```